source/common/network/io_socket_handle_impl.cc

```diff
@@ -206,8 +207,8 @@ Api::IoCallUint64Result IoSocketHandleImpl::sendmsg(const Buffer::RawSlice* slic
     message.msg_control = cbuf.begin();
     message.msg_controllen = cmsg_space;
     cmsghdr* const cmsg = CMSG_FIRSTHDR(&message);
-    RELEASE_ASSERT(cmsg != nullptr, fmt::format("cbuf with size {} is not enough, cmsghdr size {}",
-                                                sizeof(cbuf), sizeof(cmsghdr)));
+    ENVOY_BUG(cmsg != nullptr, fmt::format("cbuf with size {} is not enough, cmsghdr size {}",
```

Comment:
I think a RELEASE_ASSERT here is fine because if cmsg is null, then we'll crash below when we de-ref it.

Reply:
Good point! Reverted.